### PR TITLE
Consistency Update

### PR DESCRIPTION
**Pull request by Rustdude**
I did several small updates:
- Uniformely Renamed the code snippet tabs from "Windows", "Windows PowerShell", "PowerShell" to "Windows PowerShell"
- Uniformely renamed placeholders for addresses to "ADDRESS" (formerly sometimes named public key, very inconsistent"
- Fixed some linebreaks in PowerShell and Bash Code
- Added tabbed commands to the pchain_client documentration (to be consisitent with the "getting started" documentation)

### CODE REVIEW

File: docs/getting_started/transfer.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ tags:
 
 This guide will walk you through the process of transferring tokens on ParallelChain Mainnet / Testnet.
 
+Ich habe einen kleinen Penis
```
Comment (manngayin612) [inline docs/getting_started/transfer.md:10]: Please remove this line.

**Comment by Rustdude** [comment]
I submitted this pull request with some important changes, that remained unfixed since two weeks. Are there any issues left to resolve before the request can be merged? 
@manngayin612 noticed that I forgot to remove a test string, which is already resolved, are there any other issues open or do I need to completely reopen a pull request?

Additionally, I noticed, that the pull request failed some tests, is that the issue? Building and testing went all fine on my machine. Can you check why the automated tests failed? I noticed, that building and testing also runs fine in the GitHub Action, just the deployment to GitHub Pages fails, so I assume it could be a privilege issue? If so, maybe pull requests should not trigger a full GitHub Pages deploy script but only pass some automated tests.

**Review by manngayin612 — APPROVED** [review]
Changes has been reviewed and approved.

**Comment by manngayin612** [comment]
Hello, sorry for forgetting to merge the pull request after looking into the failing issue.

**Pull request merged**